src/editor/SluglineEditor.tsx

```diff
@@ -54,15 +58,25 @@ interface SluglineEditorProps {
   saveArticleContent: (content: Node[]) => void;
 }
 
+const SPELLCHECK_ENABLE_WAIT_MS = 500;
```

Comment:
I was going to explain how the spellchecks are hiding the underlines, but I've since realized two things:
1. This solution doesn't work that well since changing spellcheck does not change it for text that isn't being completely re-rendered.
2. As someone who has gone far sighted far before their natural time, I maybe should not take my experience as universal.

I'll just remove it, I thnk, unless you have a more creative idea.

Reply:
Removed: https://github.com/UWmathNEWS/slugline-web/pull/68/commits/3a9a04bb25b325edab7c22863b402e93cb3aaf33